### PR TITLE
Update sample/machineset.yaml to use new field and match docker version in machines.yaml.template

### PR DESCRIPTION
**Pull request by k4leung4**
**What this PR does / why we need it**:
Have docker version and os match that of the machine template

@kubernetes/kube-deploy-reviewers

### CODE REVIEW

File: sample/machineset.yaml [hunk 1]
```diff
@@ -19,9 +19,10 @@ spec:
           project: "$GCLOUD_PROJECT"
           zone: "us-central1-f"
           machineType: "n1-standard-1"
+          os: "ubuntu-1604-lts"
           image: "projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts"
```
[the remark this answers is not in the document]
Comment (k4leung4) [reply sample/machineset.yaml]: removed.

[the remark this answers is not in the document]
Comment (kcoronado) [reply sample/machineset.yaml]: It's the other way around, you want to keep the os field and get rid of the image field (sorry)

[the remark this answers is not in the document]
Comment (k4leung4) [reply sample/machineset.yaml]: trying again.

**Comment by kcoronado** [comment]
lgtm
/ok-to-test

**Comment by spew** [comment]
Hello Kenny,

Please squash these three commits down to a single commit. Also, please update the commit message with what is being updated in machineset.yaml. It could be "Update machineset.yaml sample to include the latest fields". 

Also recommend calling it "sample/machineset.yaml" instead of "machineset.yaml sample".

**Comment by k4leung4** [comment]
thanks for the suggestion.
squashed and update commit message.

**Pull request merged**